server/src/main.go

```diff
@@ -626,6 +628,17 @@ func (c MainController) NewActivistsHandler(w http.ResponseWriter, r *http.Reque
 	})
 }
 
+func (c MainController) NewActivistsPendingWorkshopHandler(w http.ResponseWriter, r *http.Request) {
+	renderPage(w, r, "activist_list", PageData{
+		PageName: "NewActivistsPendingWorkshopList",
+		Data: ActivistListData{
+			Title:       "New Activists Pending Workshop",
+			Description: "Everyone who has their first event within the given range (last 6 months by default) AND has not yet attended the workshop.",
```

Comment:
i wouldn't say "workshop" is so widely understood to mean the intro workshop. actually i didn't even know we had something called that unless it's the how-to-change-the-world-in-one-generation workshop? anway we naturally may have lots of other kinds of workshops.

```suggestion
			Description: "Everyone who has their first event within the given range (last 6 months by default) AND has not yet attended the Intro Workshop.",
```